CHANGES.rst

```diff
@@ -1,30 +1,19 @@
 Changelog
 =========
 
-6.0.2 (unreleased)
```

Comment:
no, that needs to stay... We are creating changes for a new release now.

Reply:
Okay noted